### ISSUE TITLE
How to use hprose-cpp1x ?

### ISSUE THREAD

**Issue by a12167984** [hprose/hprose-cpp1x#5]
can't find any help file for this c++, so i have  no idea of how to make a demo, how to use the downloaded file...

**Comment by laoye** [comment]
this project has not finished yet!

**Comment by timotheecour** [comment]
@laoye 
so there's currently no way to have a C++ hprose server, is that correct?
(https://github.com/hprose/hprose-cpp/issues/2 wasn't very clear)
@yesme2000

**Comment by laoye** [comment]
@timotheecour
yes, the runtime type info in c++ is too weak, so it's hard to call server function dynamically.

**Comment by xiaoyur347** [comment]
@a12167984 for build, you can follow the steps in .travis.yml(https://github.com/hprose/hprose-cpp1x/blob/master/.travis.yml). for examples, you can follow the tests like https://github.com/hprose/hprose-cpp1x/blob/master/hprose/rpc/asio/test/HttpClientTest.cpp
